contributors/guide/contributing.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Contributing to Kubernetes"
-weight: 4
-description: |
+Title: "Contributing to Kubernetes"
+Weight: 4
+Description: |
```

Comment:
Thanks! just one thing
This is metadata headers and should remain lowercase. If you take a peek at all the files in the guide, they'll all be lowercase 👍

Reply:
> Thanks! just one thing This is metadata headers and should remain lowercase. If you take a peek at all the files in the guide, they'll all be lowercase 👍

Thanks for sharing that! I have changed it back to how it was.